packages/babel-plugin/src/utils/traverse-expression/traverse-member-expression.tsx

```diff
@@ -0,0 +1,98 @@
+import traverse from '@babel/traverse';
```

Comment:
hmm I think the refactor is good :) `traverseMemberAccessPath` is exclusively used by `traverseMemberExpression`. So I think everything in `traverseMemberAccessPath` should sit under it. Thoughts on making `traverse-member-expression`and folder?

Reply:
sure, I'll do that!